corehq/apps/fixtures/models.py

```diff
@@ -409,8 +409,12 @@ def by_group(cls, group, wrap=True):
 
     @classmethod
     def by_data_type(cls, domain, data_type, bypass_cache=False):
-        data_type_id = _id_from_doc(data_type)
-        return get_fixture_items_by_data_type(domain, data_type_id, bypass_cache)
+        return cls.by_data_types(domain, [data_type], bypass_cache)
+
+    @classmethod
+    def by_data_types(cls, domain, data_types, bypass_cache=False):
+        data_type_ids = frozenset(_id_from_doc(d) for d in data_types)
```

Comment:
Ah, makes sense. I think `@memoized` _does_ require that, but quickcache is intentionally made to do its own hashing based on something closer to what we'd intuitively consider the "value" of an object. So `set(1, 2, 3)` and `set(2, 3, 1)`—separately instantiated—have the same hash. See https://github.com/dimagi/quickcache/blob/4e608301688daef7c0d026230ae7ca4a95d7b957/quickcache/quickcache_helper.py#L106 for the code if you're interested.